test/nodes/core/core/20-inject_spec.js

```diff
@@ -28,22 +28,45 @@ describe('inject node', function() {
         helper.unload();
     });
 
-    it('should inject once', function(done) {
+    it('should inject simple once', function(done) {
```

Comment:
If the default is set to 0.1 secs this should fail should't it ?

Reply:
No it should not fail because of `n.onceDelay || 0.1; // seconds` in node it is the test what happens if the node does not have that property - Node-RED does not add new properties on existing flows, right?

Reply:
myabe to rename to "should inject once without onceDelay property"

Reply:
Ah right - was missing the fact you already added 1 to the seconds... sorry !  (though mocha doesn't really like tests taking ages... but yeah guess we have to do it.

Reply:
OK, I did it because of the fix 100 msec. before

Reply:
@knolleary  done with explicit test for property